/*
 * Decompiled with CFR 0.152.
 * 
 * Could not load the following classes:
 *  net.minecraft.world.entity.player.Inventory
 */
package ic2.core.block.machine.container;

import ic2.core.block.machine.container.ContainerElectricMachine;
import ic2.core.block.machine.tileentity.TileEntityScanner;
import ic2.core.ref.Ic2ScreenHandlers;
import ic2.core.slot.SlotInvSlot;
import java.util.List;
import net.minecraft.world.entity.player.Inventory;

public class ContainerScanner
extends ContainerElectricMachine<TileEntityScanner> {
    public ContainerScanner(int n, Inventory inventory, TileEntityScanner tileEntityScanner) {
        super(Ic2ScreenHandlers.UU_SCANNER, n, inventory, tileEntityScanner, 166, 8, 43);
        this.addSlot(new SlotInvSlot(tileEntityScanner.inputSlot, 0, 55, 35));
        this.addSlot(new SlotInvSlot(tileEntityScanner.diskSlot, 0, 152, 65));
    }

    @Override
    public List<String> getNetworkedFields() {
        List<String> list = super.getNetworkedFields();
        list.add("state");
        list.add("progress");
        list.add("patternEu");
        list.add("patternUu");
        return list;
    }
}

